samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -79,7 +81,10 @@ public class StorageConfig extends MapConfig {
   public static final String STORE_RESTORE_FACTORIES = STORE_PREFIX + "%s." + RESTORE_FACTORIES_SUFFIX;
   public static final String JOB_RESTORE_FACTORIES = STORE_PREFIX + RESTORE_FACTORIES_SUFFIX;
   public static final List<String> DEFAULT_RESTORE_FACTORIES = ImmutableList.of(KAFKA_STATE_BACKEND_FACTORY);
+  public static final long DEFAULT_ROCKSDB_MAX_MANIFEST_FILE_SIZE_IN_BYTES = 1024 * 1024 * 1024L;
 
+  static final String DEFAULT_ROCKSDB_MAX_MANIFEST_FILE_SIZE =
+      STORE_PREFIX + DEFAULT_STORE_NAME + ".rocksdb.max.manifest.file.size";
```

Comment:
for the `default` config, lets not follow the same format as the config with named stores in case of `default` table name collision